modules/@apostrophecms/ui/ui/apos/components/AposSlatList.vue

```diff
@@ -132,6 +132,8 @@ export default {
       oldIndex, newIndex
     }) {
       this.next.splice(newIndex, 0, this.next.splice(oldIndex, 1)[0]);
+      // FIX: swapping the items does not trigger the watcher
+      this.next = this.next.slice();
```

Comment:
Hmm, I can't say that. I'm not sure if deep will solve it. 
I would assume deep is more CPU intensive as it should compare every property in a deep object (and we already do that in the watcher). This is a cheap check for Vue (because it fails `===`).